ca/server.go

```diff
@@ -60,7 +62,7 @@ type Server struct {
 	// the security config as a result
 	lastSeenClusterRootCA *api.RootCA
 	lastSeenExternalCAs   []*api.ExternalCA
-	secConfigMu           sync.Mutex
+	signingMu             sync.Mutex
```

Comment:
Ah ok, cool, I was hoping you might have a better name.  :)  I'm terrible at naming things, which is why I tend to end up with Java names (like `RootCAAndExternalCAMutex` or something)